src/DataAbstractionLayer/Entity/OrderActionLog/PayonePaymentOrderActionLogEntity.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PayonePayment\DataAbstractionLayer\Entity\OrderActionLog;
+
+use Shopware\Core\Checkout\Order\OrderEntity;
+use Shopware\Core\Framework\DataAbstractionLayer\Entity;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityIdTrait;
+
+class PayonePaymentOrderActionLogEntity extends Entity
```

Comment:
i don't think setters are required, because you are not using these methods.  so please remove them.

Reply:
Done